src/BaGet.Services.Mirror/Services/FakeMirrorService.cs

```diff
@@ -1,5 +1,5 @@
-﻿using System.Threading.Tasks;
-using NuGet.Versioning;
+﻿using NuGet.Versioning;
+using System.Threading.Tasks;

```

Comment:
Please keep `System` namespaces at the top (it's the .NET team's style)

Reply:
done!